lib/base/deserializers/list_response_deserializer.rb

```diff
@@ -26,6 +26,8 @@ def parse
     #   New object with account and transactions attributes
     #
     def deserialize(json)
+      return self if json.is_a?(String)
```

Comment:
@martinbarilik , wouldn't you prefer to raise an exception like 

``
raise ÌnvalidJsonResponse.new(json)
```

so you can rescue it later and display appropriate error to user?